### PR TITLE
checking for apache2 installation

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
checking whether apache2 is configured or not before checking whether it is enabled.I haven't tested it due to my present apache2 configuration.
In reference to issue #358

### CODE REVIEW

**Comment by harry-7** [comment]
@nikhilrayaprolu make sure you check your code before committing here. It seems like you missed syntax in print statement. The Python we are using is Python 3+ so change that statement [here](https://github.com/freedombox/Plinth/pull/528/files#diff-df116206a4eb33c10acba3330e695072R134)

**Comment by nikhilrayaprolu** [comment]
@harry-7 thankyou for mentioning it,Made the required changes.

File: plinth/action_utils.py [hunk 1]
```diff
@@ -121,13 +122,16 @@ def service_reload(service_name):
...
+    if spawn.find_executable('a2query'):
+        try:
+            # Don't print anything on the terminal
+            subprocess.check_output(['a2query', option_map[kind], name],
+                                    stderr=subprocess.STDOUT)
+            return True
+        except subprocess.CalledProcessError:
+            return False
+    else:
+        print("web configuration is not enabled,please reinstall apache2 web server")
```
Comment (jvalleroy) [inline plinth/action_utils.py]: I don't think we want to print anything here. This function should only return True or False (or possibly, raise an exception). I would suggest just returning False--we can assume that the web conf is disabled if there's no apache2 installed.

**Comment by nikhilrayaprolu** [comment]
@jvalleroy removed the print statement and returned False

**Comment by jvalleroy** [comment]
I have squashed these commits into just 1 commit and merged it.

**Pull request closed without merging**